lang/parser/src/cst/exp.rs

```diff
@@ -79,13 +79,15 @@ impl Exp {
 }
 
 #[derive(Debug, Clone)]
+/// either a constructor or call of a toplevel let or a variable
```

Comment:
```suggestion
/// Either a constructor or call of a toplevel let or a variable
```

nitpick (convention): in the rest of the codebase we start comments with an uppercase letter.